Source/SteveSloth/Public/MyEnemy.h

```diff
@@ -0,0 +1,41 @@
+/****************************************************************************************
+Copyright: SteveSloth
+Name: Aidan
+Script: MyEnemy.h
+Date: April 23. 2024
+Description: This is the base class for all Enemies.
+TODO:
+Known Bugs:
+TODO:
+Known Bugs:
```

Comment:
Any reason why there are duplicates of TODO and Known Bugs? if not remove.

Reply:
That was what was in the template.